packages/client/composables/useNav.ts

```diff
@@ -241,7 +242,8 @@ const useNavState = createSharedComposable((): SlidevContextNavState => {
   const router = useRouter()
 
   const currentRoute = computed(() => router.currentRoute.value)
-  const isPrintMode = computed(() => currentRoute.value.query.print !== undefined)
+  const isHandout = computed(() => currentRoute.value.query.handout !== undefined || currentRoute.value.path.startsWith('/handout'))
+  const isPrintMode = computed(() => currentRoute.value.query.print !== undefined || isHandout.value)
```

Comment:
Could we use `/print?print=handout` instead of `/handout` as the path?

Reply:
I kept it separate in order to not mess with the existing components e.g. `PrintSlide.vue`, `PrintContainer.vue` but If we want to have it rendered in one step, it would make sense to have i in one route